source/extensions/common/wasm/wasm.cc

```diff
@@ -1024,6 +1024,77 @@ uint64_t Context::getCurrentTimeNanoseconds() {
       .count();
 }
 
+// TODO(kyessenov) this is available but not exposed in the upstream, need to correct this
```

Comment:
Do we have an issue for this?  If not let's create one and reference it here.

Reply:
Done.